ui/src/notebooks/pipes/Query/ResizerHeader.tsx

```diff
@@ -0,0 +1,66 @@
+// Libraries
+import React, {FC, RefObject} from 'react'
+import classnames from 'classnames'
+
+// Components
+import {Icon, IconFont} from '@influxdata/clockface'
+
+// Types
+import {ResultsVisibility} from 'src/notebooks/pipes/Query'
+
+interface Props {
+  visibility: ResultsVisibility
+  onUpdateVisibility: (visibility: ResultsVisibility) => void
+  onStartDrag: () => void
+  resizingEnabled: boolean
+  dragHandleRef: RefObject<HTMLDivElement>
+}
+
+const ResultsHeader: FC<Props> = ({
+  visibility,
```

Comment:
i'm conflicted about leaving this because it breaks the pattern, but lets just leave it until we add another property from data